devops/resources/missing-newrelic-alerts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd $WORKSPACE/configuration/util/jenkins/
+
+pip install -r check_celery_progress/requirements.txt
```

Comment:
I would probably generate your own requirements.txt and put your script and requirements directory in their own directory instead of using the check_celery_progress/requirements.txt

This is so that if you want to delete one of the scripts later you dont have to pull them apart.